lib/Drush/Drush.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @file
+ * Definition of Drush\Drush.
+ */
+
+namespace Drush;
+
+/**
+ * The Drush API.
+ */
+class Drush {
+	static $commandfiles_cache;
+	
+	/**
+	 * Run a drush command in a new process
+	 */
+	static function drush($site, $command_name, $commandline_args = array(), $commandline_options = array(), $backend_options = TRUE) {
```

Comment:
Is this method and invoke() in use anywhere? In general, I'm a little hesitant to have a Drush class as that can become a grab bag of unrelated methods.